### PR TITLE
Docu fix

### PR DESCRIPTION
**Pull request by Piankero**
## Basics

These points need to be fulfilled for every PR:

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy.
- [ ] The PR is rebased with current master.

If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you.
But we cannot accept PRs that do not fulfill the basics.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned every code not directly written by me in [THIRD-PARTY-LICENSES](doc/THIRD-PARTY-LICENSES)

## Review

Reviewers will usually check the following:

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

If you are already Elektra developer:

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the basics are fulfilled** and you also
  say that everything is ready to be merged.

### CODE REVIEW

File: doc/tutorials/plugins.md [hunk 1]
```diff
@@ -356,15 +356,17 @@ and write each key as its own line in the file. Since we don't care about the na
 the value of `keyString` for each `Key` as a new line in the file. That's it. Now, each time the mounted `KeySet` is modified, `elektraPluginSet` will
 be called and the mounted file will be updated.
 
-#### `ELEKTRA_SET_ERROR`
+#### `ELEKTRA_SET_<CONCRETE>_ERROR`
```
Comment (markus2330) [inline doc/tutorials/plugins.md]: better word than `<CONCRETE>`? Which are available?

File: doc/tutorials/plugins.md [hunk 2]
```diff
@@ -419,8 +421,9 @@ keyNew ("system/elektra/modules/" ELEKTRA_PLUGIN_NAME "/exports/checkconf", KEY_
 ```
 
 Within the `checkconf` function all of the plugin configuration values should be validated.
-Errors should be reported via Elektra's error handling mechanism (see section [ELEKTRA_SET_ERROR](#elektra_set_error) for further details).
-If `checkconf` encounters a configuration value, that is not strictly invalid but can not be parsed by the plugin (e.g. a parameter which is not part of the plugin configuration), then a warning should be appended to `errorKey`, using `ELEKTRA_ADD_WARNING`.
+Errors should be reported via Elektra's error handling mechanism (see section [ELEKTRA*SET*<CONCRETE>\_ERROR](#elektra_set_concrete_error) for further details).
+If `checkconf` encounters a configuration value, that is not strictly invalid but can not be parsed by the plugin (e.g. a parameter which is not part of the plugin configuration),
+then a warning should be appended to `errorKey`, using `ELEKTRA_ADD_<CONCRETE>_WARNING`.
```
Comment (markus2330) [inline doc/tutorials/plugins.md]: Not mentioned that there is also `ERRORF`?

File: doc/tutorials/plugins.md (hunk 1 above)
Comment (Piankero) [reply doc/tutorials/plugins.md]: I am also not quite sure and found <CONCRETE> the best but what about:
`ELEKTRA_SET_..._ERROR`, `ELEKTRA_SET_<TYPE>_ERROR`,  `ELEKTRA_SET_<CONCRETE_TYPE>_ERROR` ...?

File: doc/tutorials/plugins.md (hunk 2 above)
Comment (Piankero) [reply doc/tutorials/plugins.md]: It was not documented before but I may add it

File: doc/tutorials/plugins.md (hunk 1 above)
Comment (markus2330) [reply doc/tutorials/plugins.md]: Yes, maybe "concrete type". But you should define what a concrete or abstract type is.

File: doc/tutorials/plugins.md (hunk 2 above)
Comment (markus2330) [reply doc/tutorials/plugins.md]: What do you mean with "may"?

**Comment by markus2330** [comment]
Still two open points and Jenkins did not built the PR.

File: doc/tutorials/plugins.md (hunk 2 above)
Comment (Piankero) [reply doc/tutorials/plugins.md]: I "can" add it is more appropriate. I added the text already

Comment (markus2330) [reply doc/tutorials/plugins.md]: Thank you!

Comment (markus2330) [reply doc/tutorials/plugins.md]: It does not really explain how to use ERRORF or WARNINGF. Furthermore, at some places there is still `<CONCRETE>` used. I merge it nevertheless, as it is a small improvement.

**Pull request merged**